_search-plugins/knn/api.md

```diff
@@ -291,32 +334,28 @@ DELETE /_plugins/_knn/models/{model_id}
 }
 ```
 
-## Train Model
-Introduced 1.2
-{: .label .label-purple }
+## Train model
 
-Create and train a model that can be used for initializing k-NN native library indexes during indexing. This API will 
-pull training data from a `knn_vector` field in a training index and then create and train a model and then serialize it
-to the model system index. Training data must match the dimension passed into the body of the request. This request 
-will return when training begins. To monitor the state of the model, use the [Get model API](#get-model).  
+Create and train a model that can be used for initializing k-NN native library indexes during indexing. This API pulls training data from a `knn_vector` field in a training index, creates and trains a model, and then serializes it to the model system index. Training data must match the dimension passed in the request body. This request returns when training begins. To monitor the model's state, use the [Get model API](#get-model).  
 
-Query Parameter |  Description
+Query parameter |  Description
 :--- | :---
-`model_id` | (Optional) The id of the fetched model. If not specified, a random id will be generated.
-`node_id` | (Optional) Preferred node to execute training. If set, this node will be used to perform training if it is deemed to be capable.
+`model_id` | (Optional) The unique identifier of the fetched model. If not specified, a random id is generated.
```

Comment:
Global: Is this how we normally include/format "(Optional)"? I feel like I normally see it in a separate column or at the end.

Reply:
Global changes made to include at end.